yapf/yapflib/file_resources.py

```diff
@@ -23,6 +23,27 @@
 from lib2to3.pgen2 import tokenize
 
 from yapf.yapflib import py3compat
+from yapf.yapflib import style
+
+
+LOCAL_STYLE = '.style.yapf'
```

Comment:
If the comment is on the PR/files page they remain as "outdated diff" (even if you force push). Not sure whether the comments on google/yapf/commit  remain after a force push, but it looks like if they're on commits on your fork they are definitely not shown.

This comment was @eliben, so not sure what the intention was (I was misremembering the line).